tensorbay/client/gas.py

```diff
@@ -180,6 +180,15 @@ def list_auth_storage_configs(self) -> PagingList[Dict[str, Any]]:
             128,
         )
 
+    def delete_storage_config(self, name: str) -> None:
+        """Delete a storage config in TensorBay.
+
+        Arguments:
+            name: Name of the storage config, unique for a team.
+
+        """
+        self._client.open_api_do("DELETE", "auth-storage-configs", json={"name": name})
```

Comment:
Should us update the OpenAPI name from `"auth-storage-configs" to "storage-configs"`